src/card/CardThunks.ts

```diff
@@ -157,7 +157,7 @@ export const updateReportSettingThunk = (index, setting, value) => (dispatch: an
 
```

Comment:
Generic comment:
Reloading a default dashboard with the new ID-based reports lets some reports stay in the state. (i.e. the state is never really cleaned up when doing a dashboard load).
Not sure where this happens or how to fix it though :)